packages/cli-common/resources/templates/template-project/admin/index.html

```diff
@@ -2,7 +2,7 @@
 <html>
 	<head>
 		<meta charset="utf-8">
-		<meta name="viewport" content="width=device-width">
+		<meta name="viewport" content="width=device-width, initial-scale=1, viewport-fit=cover"/>
```

Comment:
@JanTvrdik any particular reason you omitted this when implementing layout changes in https://github.com/contember/engine/commit/0806c7d5ab0b1bfcc11259dfcd5f51ab57603b51?

Reply:
AFAIK `initial-scale=1` is useful only for old Safari. And I don't have phone with a notch to test if `viewport-fit=cover` does anything useful.